Firestore/Source/API/FIRLoadBundleTask.mm

```diff
@@ -85,25 +67,20 @@ - (instancetype)initWithTask:(std::shared_ptr<api::LoadBundleTask>)task {
   return self;
 }
 
-- (FIRLoadBundleHandle)observeState:(FIRLoadBundleTaskState)state
-                            handler:(void (^)(FIRLoadBundleTaskProgress *progress))handler {
+- (FIRLoadBundleHandle)observeWithHandler:(void (^)(FIRLoadBundleTaskProgress *progress))handler {
   if (!handler) {
-    return nil;
+    return NSIntegerMin;
```

Comment:
Hmm, should you use `ThrowInvalidArgument` instead? Other Objective-C public classes seem to do that, though feel free to disagree.

Reply:
Done.